code/index.html

```diff
@@ -2,21 +2,75 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Yummy food</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
+  
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+      <img class=logo src="./images/logo.png"> <!-- Some kind of logo and navbar -->
+   
+      <nav>
+        <a class="order-1" href="#">Start</a>
+        <a class="order-2" href="#">Recepies</a>
+        <a class="order-3" href="#">About</a>
+        <a class="order-4" href="#">Contact</a>
+      </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+    <section class="big-news">  <!-- The big news content -->
+      <img src="./images/food_1.jpg" alt="breakfast_bowl" style="width:100%">
+      <div class="big-news-text"> 
+      <h3>Yummy food recepies and inspiration</h3>
+       <p>On this site you will find recipes and a lot of inspiration for breakfast, lunch, dinner, snacks and so on.</p>
+       <a href="#">Read more</a>
+       </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <section class="other-news"> <!-- The other news  -->
+      
+      <div class="news1">
+        <img src="./images/food_2.jpg" alt="food2" style="width:100%">
+        <a href="#"><h3>Breakfast</h3></a>
+      </div>
+
+      <div class="news2">
+        <img src="./images/food_3.jpg" alt="food3" style="width:100%">
+        <a href="#"><h3>Lunch</h3></a>
+       </div>
```

Comment:
Take a look at the indentation on all news, put all the child elements on a new line:
```
<div class="news2">
  <img src="./images/food_3.jpg" alt="food3" style="width:100%">
  <a href="#">
    <h3>Lunch</h3>
  </a>
</div>
```